src/playbooks/check/check.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Run preflight checks
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Run hostname check script
+      command: "{{ playbook_dir }}/../../roles/hostname_check/files/foreman-hostname-check"
```

Comment:
Instead of a full-blown script, could we use `assert` with the `ansible_fqdn` fact?

Something like:
```
- name: check hostname
  ansible.builtin.assert:
    that:
      - "'_' not in ansible_fqdn"
      - "'.' in ansible_fqdn"
```
etc?

Reply:
Yes, I tried doing that. I can't clearly understand if it is working.